src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-repositorySlug.html

```diff
@@ -1,4 +1,4 @@
 <div>
-    Enter the repository slug. The slug can be found in the URL after the repos/ line. **Note** this may be different
-    to the *repo name* listed on the page at your.bitbucket.url.com/project/YOURPROJECTKEY/repos/
+    This is the repository name that is used in the repository URL. For example, for the repository URL
```

Comment:
I don't think we should call the repo slug the 'repository name' because that can be confusing.

Reply:
If the field is called "repository slug" or similar and we wanted to explain it here perhaps we could rephrase this to say something like "The slug is the URL-safe version of the repository name."?

Reply:
Following what the [BBC docs say about repository slugs ](https://confluence.atlassian.com/bitbucket/what-is-a-slug-224395839.html) I've changed this to:
> This is the URL-friendly version of the repository name. 
Thanks